astronomer/providers/google/cloud/example_dags/example_bigquery_sensors.py

```diff
@@ -0,0 +1,97 @@
+"""Example Airflow DAG for Google BigQuery Sensors."""
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+    BigQueryInsertJobOperator,
+)
+from airflow.providers.google.cloud.sensors.bigquery import (
+    BigQueryTablePartitionExistenceSensor,
+)
+
+from astronomer.providers.google.cloud.sensors.bigquery import (
+    BigQueryTableExistenceSensorAsync,
+)
+
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "astronomer-airflow-providers")
+DATASET_NAME = os.environ.get("GCP_BIGQUERY_DATASET_NAME", "astro_dataset")
+GCP_CONN_ID = os.environ.get("GCP_CONN_ID", "google_cloud_default")
+LOCATION = os.environ.get("GCP_LOCATION", "us")
+
+TABLE_NAME = os.environ.get("TABLE_NAME", "partitioned_table")
+INSERT_DATE = datetime.now().strftime("%Y-%m-%d")
+
+PARTITION_NAME = "{{ ds_nodash }}"
+
+INSERT_ROWS_QUERY = f"INSERT {DATASET_NAME}.{TABLE_NAME} VALUES " "(42, '{{ ds }}')"
+
+SCHEMA = [
+    {"name": "value", "type": "INTEGER", "mode": "REQUIRED"},
+    {"name": "ds", "type": "DATE", "mode": "NULLABLE"},
+]
+
+dag_id = "example_bigquery_sensors"
+
+with models.DAG(
```

Comment:
nit:  I prefer `from airflow.models.dag import DAG` or `from airflow import DAG` then use it with `with DAG(...)` because first one is more informative and tell reader the exact path of `DAG` and second one is more clean.

Reply:
@bharanidharan14 can you make this change so that we can merge it to main please

Reply:
Fixed